packages/components/psammead-social-embed/src/Canonical/index.jsx

```diff
@@ -47,6 +47,11 @@ export const providers = {
         window.twttr.widgets.load();
       }
     },
+    onSdkLoad: onRender => {
```

Comment:
As part of the code review it would be good to know by reviewers if `onRender` is too generic or if this prop should be renamed to be Twitter specific?